config/collector.js

```diff
@@ -56,6 +56,21 @@ const createScanConfig = (dirs, cwdPattern) => ({
             onlyFiles: false,
             onlyDirectories: true
         }
+    },
+
+    twigs: {
+        dirs,
+        patterns: [
+            '**/Theme/default/views/*',
+            '**/Theme/default/templates/*',
```

Comment:
I think that in this config path collector will always parse project and core levels. Please recheck and change 
`${cwdPattern}Theme/default/views/*'
....